src/pages/result/utils/filter.util.ts

```diff
@@ -4,9 +4,36 @@ import { PetProfile } from '../../../shared/interfaces';
 export const getFiltredPets = (pets: PetProfile[], filterValues: FilterValues): PetProfile[] => {
   const notation = 10;
 
-  const getComparisonResult = (petValue: string, userInputValue: string): boolean => {
+  const getComplicatedComparisonResult = (petValue: any, userInputValue: any): boolean => {
+    const low = 2,
+      average = 3,
+      high = 4;
+
+    if (typeof userInputValue === 'boolean') {
+      return petValue >= average;
+    } else if (typeof userInputValue === 'string') {
+      switch (userInputValue) {
+        case 'low':
+          return petValue <= low;
+        case 'averege':
+          return petValue === average;
```

Comment:
Is this correct?
averege != average

Reply:
Yep, this is correct. 
And average is a variable which contains a number '3';
I know that I have here a spelling mistake, 'averege' is not correct, but this value I took from Pasha so I'll change it when Pasha changes it on his page. ))